src/client/src/selectors/postGenerationSelector.ts

```diff
@@ -81,13 +81,13 @@ const servicesToDeploy = (
 ): IAzureServiceStatus => {
   return {
     cosmosdb: {
-      title: messages.cosmosTitle,
+      title: azureMessages.cosmosTitle,
```

Comment:
I am just a little confused here. The azure messages are coming from the mockData. Are these always supposed to come from mockData (I mean even when we are running in extension)? 
If yes, I think we should store this data somewhere else rather than mockData since it is misleading (the data is not actually mock in that case, maybe staticData)

Reply:
The client gets all the data for azure services page from the mockData (maybe it was supposed to be moved to be retrieved from extension later?). I agree that we should move it or rename it

Reply:
Yup - will do it in another pr